Mathlib/Analysis/SpecialFunctions/Pow/Real.lean

```diff
@@ -58,15 +59,31 @@ theorem rpow_def_of_pos {x : ℝ} (hx : 0 < x) (y : ℝ) : x ^ y = exp (log x *
 theorem exp_mul (x y : ℝ) : exp (x * y) = exp x ^ y := by rw [rpow_def_of_pos (exp_pos _), log_exp]
 #align real.exp_mul Real.exp_mul
 
+@[simp, norm_cast]
+theorem rpow_int_cast (x : ℝ) (n : ℤ) : x ^ (n : ℝ) = x ^ n := by
+  simp only [rpow_def, ← Complex.ofReal_zpow, Complex.cpow_int_cast, Complex.ofReal_int_cast,
+    Complex.ofReal_re]
+#align real.rpow_int_cast Real.rpow_int_cast
+
+@[simp, norm_cast]
+theorem rpow_nat_cast (x : ℝ) (n : ℕ) : x ^ (n : ℝ) = x ^ n := by simpa using rpow_int_cast x n
```

Comment:
Whats with these being `nat_cast` vs `natCast` elsewhere?

Reply:
We've hardly decided on a convention. I'm personally in favor of `natCast` (and the poll a few months back agrees with me) but most lemmas have been ported as `nat_cast`. This lemma is just moving, so I'm not changing its name. However I'm naming the new ones using `natCast`.